tools/unitctl/Cargo.lock

```diff
@@ -26,6 +26,21 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "android-tzdata"
+version = "0.1.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "e999941b234f3131b00bc13c22d06e8c5ff726d1b6318ac7eb276997bbb4fef0"
+
+[[package]]
+name = "android_system_properties"
+version = "0.1.5"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "819e7219dbd41043ac279b19830f2efc897156490d7fd6ea916720117ee66311"
+dependencies = [
+ "libc",
+]
+
```

Comment:
Not sure why we need Android stuff for docker?

Reply:
Its being brought in by chrono, a library which is used in serde_with (a serialization helper library). It looks like the author could just turn off these imports without consequence since they arent using the features in chrono which use them... so I left a ticket. I will keep an eye on that but I am not sure I can pull it out as part of this changeset.